apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -81,10 +80,10 @@ defmodule Aecore.Chain.Worker do
     GenServer.call(__MODULE__, :top_height)
   end
 
-  @spec get_block_by_bech32_hash(String.t()) :: Block.t() | {:error, binary()}
-  def get_block_by_bech32_hash(hash) do
-    decoded_hash = Bits.bech32_decode(hash)
-    get_block(decoded_hash)
+  @spec get_block_by_base58_hash(String.t()) :: Block.t()
+  def get_block_by_base58_hash(hash) do
+    decoded_hash = Header.base58c_decode(hash)
+    GenServer.call(__MODULE__, {:get_block_info_from_memory_unsafe, decoded_hash})
```

Comment:
why doesn't this use the save call any more?

Reply:
I have no idea why did is changed, we'll fix this